src/dipali/Assignment19.java

```diff
@@ -0,0 +1,33 @@
+package dipali;
+
+public class Assignment19 {
+	
+	public String getDetails(String input) {
+		int asciiUpper=0;
+		int asciilower=0;
+		if(input!=null) {
+			int len=input.length();
+			
+			int ascii=0;
+			for(int cnt=0;cnt<len;cnt++) {
+				ascii =ascii+(int) input.charAt(cnt);
+				if(ascii>65 || ascii==65) {
+					asciiUpper =ascii+(int) input.charAt(cnt);
```

Comment:
Here, You are doing ascii value sum, as per the requirement we need to count upper case and lower count.

in this case lower case sum always higher than the upper case char reason lower case start from 97 and upper starts with 65

Reply:
You should use Character.isUpperase(ch) to fulfil requirement